harmony-api/src/repos/followRepo.ts

```diff
@@ -0,0 +1,65 @@
+import prisma from '../../prisma/prisma'
+import { HttpError } from '../models/error/httpError'
+import { Follow } from '../models/follow'
+
+export const followUser = async (followInfo: Follow): Promise<Follow> => {
+  try {
+    const followResult = await prisma.follow.create({
+      data: {
+        followingId: followInfo.followingId,
+        followerId: followInfo.followerId,
+      },
+    })
+
+    return {
+      followingId: followResult.followingId,
+      followerId: followResult.followerId,
+    }
+  } catch (e) {
```

Comment:
After this sprint lets start type checking our prisma errors so that if the DB is down we wont be getting the wrong error response like this